### PR TITLE
Feature/nfc/acr122 

### PR DESCRIPTION
**Pull request by TsengWen**
USCC Attendance System Down~
$./uscc_AttendanceSystem.py help

### CODE REVIEW

File: uscc_AttendanceSystem.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+	"mute":[0xFF, 0x00, 0x52, 0x00, 0x00],
+	"unmute":[0xFF, 0x00, 0x52, 0xFF, 0x00],
+	"getuid":[0xFF, 0xCA, 0x00, 0x00, 0x00],
+}
+
+COMMAND = cmdMap.get(cmd, cmd)
+
+
+## member dictionary
+member = {}
```
Comment (iankuan) [inline uscc_AttendanceSystem.py]: Hard code

File: uscc_AttendanceSystem.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+def member():
+
+	global member
+	member = {}
```
Comment (iankuan) [inline uscc_AttendanceSystem.py]: Ditto

File: uscc_AttendanceSystem.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+cmd = sys.argv[1]
+
+if cmd == "help":
+	print "usage: uscc_AttendanceSystem.py <command>\nList of available commands: help, start, getuid, mute, unmute."
+	print "Before executing command, make sure that \"member.txt\" has been sign in."
+	print "\thelp\tShow this help page"
+	print "\tstart\tStart running USCC Attendence System."
+	print "\tmute\tDisable beep sound when card is tagged."
+	print "\tunmute\tEnable beep sound when card is tagged."
+	print "\tgetuid\tPrint UID of the tagged card."
```
Comment (iankuan) [inline uscc_AttendanceSystem.py]: It doesn't work in Python3, does it?

**Comment by iankuan** [comment]
Typo? Down?
Furthermore, It doesn't work in Python3, does it? Please see the requirement in README.

File: uscc_AttendanceSystem.py (hunk 1 above)
Comment (TsengWen) [reply uscc_AttendanceSystem.py]: I used member.txt to build the dictionary.

**Comment by TsengWen** [comment]
Fix python3 problem

**Comment by iankuan** [comment]
Please rebase your commits message first. I would review your code until you finish that.

**Pull request closed without merging**